Server/JokesOnYou.Web.Api/DTOs/JokeUpdateDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.DTOs
+{
+    // Does not update joke's id, likes, author, uploaddate, or dislikes. Just Premise and Punchline
+    public class JokeUpdateDTO
```

Comment:
What is the naming convention? I saw some classes named ...DTO and some ...Dto

Reply:
Yea my bad.... I started with all capital like these and then remembered:
`For abreviations, use Pascal Case as well. For example Bmi` [Found in wiki](https://github.com/Code2Gether-Discord/JokesOnYou/wiki/Coding-Standards).